lib/galaxy/datatypes/mothur.py

```diff
@@ -16,18 +16,24 @@ class Otu(Text):
     file_ext = 'mothur.otu'
     MetadataElement(name="columns", default=0, desc="Number of columns", readonly=True, visible=True, no_value=0)
     MetadataElement(name="labels", default=[], desc="Label Names", readonly=True, visible=True, no_value=[])
+    MetadataElement(name="otulabels", default=[], desc="OTU Names", readonly=True, visible=True, no_value=[])
 
     def __init__(self, **kwd):
         Text.__init__(self, **kwd)
 
     def set_meta(self, dataset, overwrite=True, **kwd):
         if dataset.has_data():
             label_names = set()
+            otulabel_names = set()
```

Comment:
maybe there are duplicate headers added below? (that would be eliminated by `set`)